src/client/java/minicraft/item/FurnitureItem.java

```diff
@@ -108,6 +141,9 @@ public boolean isDepleted() {
 	public @NotNull FurnitureItem copy() {
 		// in case the item is a spawner, it will use the sprite position (sx, sy)
 		// instead if it is not, the constructor will obtain said sprite
-		return new FurnitureItem(furniture.copy());
+		if (!isRideable)
+			return new FurnitureItem(furniture.copy());
+		else
+			return new FurnitureItem(rideable);
```

Comment:
.copy()

Reply:
There is no real copy class for rideable